frontend/src/pages/ActivityReport/Pages/components/ObjectiveTta.js

```diff
@@ -32,22 +37,36 @@ export default function ObjectiveTta(
 
   return (
     <FormGroup error={error.props.children}>
-      <Label className="ttahub-objective-tta" error={error.props.children}>
-        TTA provided
-        {' '}
-        <Req announce />
-        {error}
-        <div className="smart-hub--text-area__resize-vertical margin-top-1">
-          <input type="hidden" name={inputName} value={ttaProvided} />
-          <RichEditor
-            ariaLabel="TTA provided for objective, required"
-            defaultValue={ttaProvided}
-            value={ttaProvided}
-            onChange={onChangeTTA}
-            onBlur={validateTta}
-          />
-        </div>
-      </Label>
+      <Drawer
+        triggerRef={drawerTriggerRef}
+        stickyHeader
+        stickyFooter
+        title="Get help writing TTA provided"
+      >
+        <ContentFromFeedByTag className="ttahub-drawer--objective-tta-provided" tagName="ttahub-tta-provided" contentSelector="table" />
```

Comment:
You can remove contentSelector="table" it's just causing an unnecessary pass through the fetched-in DOM

see console:
```
No content element found with selector table displaying entire contents instead